src/pudl/workspace/datastore.py

```diff
@@ -463,25 +457,19 @@ def parse_command_line():
     parser.add_argument(
         "--dataset",
         help="Download the specified dataset only. See below for available options. "
-        "The default is to download all, which may take an hour or more."
-        "speed.",
+        "The default is to download all datasets, which may take hours depending on "
+        "network speed.",
     )
     parser.add_argument(
         "--pudl_in",
-        help="Override pudl_in directory, defaults to setting in ~/.pudl.yml",
+        help="Input directory to use, overridng the $PUDL_INPUT environment variable.",
```

Comment:
I think there are some other lingering references to `.pudl.yml` floating around that we should chase down now that we've switched over to using `$PUDL_INPUT` and `$PUDL_OUTPUT` entirely, but that's for another PR.

Reply:
I don't know. If anywhere I think it would be in the tests. I grepped a bit and didn't see it anywhere.

Reply:
I took a look around and also only see "manually run this script to cache datastore locally." So my vote is to use the env vars completely.

Reply:
Okay, I'll remove this and see if everything keeps working!

Reply:
But more generally I think @rousik agreed to take on the "scour the repo for remaining references to `.pudl.yml` and friends" task.